### PR TITLE
NEW POWERUP! - Add Life Powerup

### PR DESCRIPTION
**Pull request by s0mbieZ**
This adds a new purple powerup that adds a life to the player!

TODO:
@elijahscherz you need to add a sound when the player picks up the
powerup, I can’t figure out how to do that.

### CODE REVIEW

**Comment by s0mbieZ** [comment]
@elijahscherz idk why its 707 files, the only ones I changed are `GameManager.cs` and `Powerup.cs`  The rest must be important Unity stuff

File: Assets/Scripts/GameManager.cs [hunk 1]
```diff
@@ -10,6 +10,7 @@ public class GameManager : MonoBehaviour {
     public GameObject bulletPowerupPrefab;
     public GameObject shipControlPowerupPrefab;
     public GameObject doubleShotPowerupPrefab;
+	public GameObject addLifePowerupPrefab;
```
Comment (elijahscherz) [inline Assets/Scripts/GameManager.cs]:
``` c#
public GameObject addLifePowerupPrefab;
```

Make this line not indented or spaced or whatever it is, I cannot edit your edits to the document lol, but I can comment.

**Comment by elijahscherz** [comment]
@s0mbieZ 
Oh, I know why your commit is huge, you added the library files. Which I hadn't added yet. Any way you can remove them from the commit?

Also, there are images in this commit correct?

File: Assets/Scripts/GameManager.cs (hunk 1 above)
Comment (s0mbieZ) [inline Assets/Scripts/GameManager.cs]: ok i fixed it i think... I think stupid Github is doing this crap because I'm not putting in dumb spaces...

**Pull request closed without merging**